indy_node/server/request_handlers/read_req_handlers/get_attribute_handler.py

```diff
@@ -1,74 +1,108 @@
-from indy_node.server.request_handlers.domain_req_handlers.attribute_handler import AttributeHandler
-from plenum.server.request_handlers.handler_interfaces.read_request_handler import ReadRequestHandler
-from indy_common.constants import ATTRIB, GET_ATTR
-from indy_node.server.request_handlers.utils import validate_attrib_keys
-from plenum.common.constants import RAW, ENC, HASH, TARGET_NYM, DOMAIN_LEDGER_ID
+from typing import Any, Mapping
+
+from plenum.common.constants import DOMAIN_LEDGER_ID, ENC, HASH, RAW, TARGET_NYM
 from plenum.common.exceptions import InvalidClientRequest
 from plenum.common.request import Request
 from plenum.common.txn_util import get_request_data
 from plenum.server.database_manager import DatabaseManager
+from plenum.server.request_handlers.utils import decode_state_value
 from stp_core.common.log import getlogger
 
+from indy_common.constants import ATTRIB, GET_ATTR, TIMESTAMP
+from indy_node.server.request_handlers.domain_req_handlers.attribute_handler import (
+    AttributeHandler,
+)
+from indy_node.server.request_handlers.read_req_handlers.version_read_request_handler import (
+    VersionReadRequestHandler,
+)
+from indy_node.server.request_handlers.utils import validate_attrib_keys
+
 logger = getlogger()
 
 
-class GetAttributeHandler(ReadRequestHandler):
+class GetAttributeHandler(VersionReadRequestHandler):
```

Comment:
The `GetAttributeHandler` received significant refactors while adding the ability to resolve by seqNo or timestamp. Tests are passing and cover this handler well but we'll be on alert for any unexpected behavior.